homeassistant/requirements.py

```diff
@@ -15,12 +16,44 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+class RequirementsNotFound(Exception):
```

Comment:
Should we inherit from `HomeAssistantError`?

Reply:
Hehe, would have avoided my missed testcase I just pushed a fix for.

Yea i think it could make sense.